packages/walletconnect/src/provider.ts

```diff
@@ -134,19 +134,8 @@ export class WalletConnectProvider extends SignerProvider {
       this.updateNamespace(this.session.namespaces)
     } else {
       this.updateNamespace(this.session.namespaces)
-      await this.ping(this.session.topic)
```

Comment:
I want to remove ping for two reasons:
1. Before the ping request returns, the dApp cannot proceed to the next step, which is not very user-friendly, and the ETH walletconnect provider will not send the ping request either
2. Even if the wallet is not open when the dApp sends the request, the request will be saved to the walletconnect relayer server. Once the wallet is open and connected to the relay, the request will be received.